### PR TITLE
login page integration

### PR DESCRIPTION
**Pull request by Afinjuomo**
I integrated the login page and also made some changes on the server file

### CODE REVIEW

File: apps/Client/src/pages/Login/LoginPage.tsx [hunk 1]
```diff
@@ -1,18 +1,44 @@
...
   const { register, handleSubmit } = useForm();
 
-  const onSubmit = (data: unknown) => {
-    // Handle the logic for requesting a rider using the form data
-    console.log('Requesting rider with data:', data);
-    // Add additional logic such as sending a request to your backend
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const onSubmit = async (data: any) => {
+    try {
+      const response = await fetch('http://localhost:5006/api/v1/riders/login', {
```
Comment (Don-christo) [inline apps/Client/src/pages/Login/LoginPage.tsx:16]: This is a hard-coded value. What if my port is not 5006, this won't work. I think you should add these to the env file, this way, it works with anyone who works with the code. What do you think?

File: apps/Client/src/pages/Login/LoginPage.tsx [hunk 2]
```diff
@@ -1,18 +1,44 @@
...
 
 const LoginPage: React.FC = () => {
   const { register, handleSubmit } = useForm();
 
-  const onSubmit = (data: unknown) => {
-    // Handle the logic for requesting a rider using the form data
-    console.log('Requesting rider with data:', data);
-    // Add additional logic such as sending a request to your backend
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const onSubmit = async (data: any) => {
```
Comment (Don-christo) [inline apps/Client/src/pages/Login/LoginPage.tsx:14]: Using any type is not advisable when using typescript. Try finding a way to fix the code without using any type. I think Chat can help out with it.

File: apps/Client/src/pages/Login/LoginPage.tsx [hunk 3]
```diff
@@ -1,18 +1,44 @@
...
 
-// Import your logo and button here
 import riderPhoto from "../../pages/Auth/Images/image 4.png";
 import Button from '../../components/Button';
 import riderLogo from "../../pages/Auth/Images/Logo.png";
 
 const LoginPage: React.FC = () => {
   const { register, handleSubmit } = useForm();
 
-  const onSubmit = (data: unknown) => {
```
Comment (Don-christo) [inline apps/Client/src/pages/Login/LoginPage.tsx:12]: Using unknown type is not advisable when using typescript. Try finding a way to fix the code without using unknown type. I think Chat can help out with it.

File: apps/server/src/app.ts [hunk 4]
```diff
@@ -16,27 +16,10 @@ const app = express();
...
-  ENV.FE_BASE_URL as string,
-  // CORS allow use of swagger on local environment
-  ENV.IS_PROD ? "" : `http://localhost:${port}`,
-].filter(Boolean);
 
-const corsOptions: cors.CorsOptions = {
-  origin: allowedOrigins,
-  methods: ["GET", "POST", "PUT", "DELETE", "OPTIONS"],
-  credentials: true,
-};
```
Comment (Don-christo) [inline apps/server/src/app.ts:29]: Why did you delete these cors methods??

File: apps/server/src/app.ts [hunk 5]
```diff
@@ -16,27 +16,10 @@ const app = express();
...
 
-app.use(cors(corsOptions));
-app.use((_req: Request, res: Response, next: NextFunction) => {
-  res.header(
-    "Access-Control-Allow-Headers",
-    "Origin, X-Requested-With, Content-Type, Accept"
-  );
-  res.header("Referrer-Policy", "no-referrer-when-downgrade"); // this header is needed when using http and not https
-  next();
-});
```
Comment (Don-christo) [inline apps/server/src/app.ts:39]: ... why?

**Review by Don-christo — CHANGES_REQUESTED** [review]

File: apps/Client/src/pages/Login/LoginPage.tsx (hunk 1 above)
Comment (Afinjuomo) [reply apps/Client/src/pages/Login/LoginPage.tsx:16]: that's fine

File: apps/server/src/app.ts (hunk 4 above)
Comment (Afinjuomo) [reply apps/server/src/app.ts:29]: It was redundant, not allowing me to access another page

**Comment by Don-christo** [comment]
The login page has been integrated.
